README.md

```diff
@@ -8,9 +8,9 @@ Status](https://travis-ci.org/ccap/purescript-ccap-codegen.svg)](https://travis-
 ### Testing
 
 ```sh
-$ pulp run -- -f county.tmpl -m pretty
-$ pulp run -- -f county.tmpl -m test
-$ pulp run -- -f county.tmpl -m purs > County.purs
-$ pulp run -- -f county.tmpl -m scala > County.scala
-$ pulp run -- -f county.tmpl -m show
+pulp run -- -m pretty -p unused county.tmpl
+pulp run -- -m test -p unused county.tmpl
+pulp run -- -p Ccap.Cc -m purs county.tmpl > County.purs
+pulp run -- -p gov.wicourts.cc -m scala > county.tmpl County.scala
```

Comment:
`pulp run -- -p gov.wicourts.cc -m scala county.tmpl > County.scala`

(Placement of `>`)